package/main/default/objects/TV_Serie__c/fields/Status__c.field-meta.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
What for do you need Status on TV Serie?

Reply:
Done